src/Famix-Traits/FamixTWithThrownExceptions.trait.st

```diff
@@ -32,8 +32,6 @@ FamixTWithThrownExceptions >> thrownExceptions [
 
 { #category : #accessing }
 FamixTWithThrownExceptions >> thrownExceptions: anObject [
-
-	<generated>
-	thrownExceptions value: anObject
+	thrownExceptions add: anObject
```

Comment:
This should be a new method #addThrownException:: and not change the implementation of #thrownExceptions: